apollo-router/src/apollo_telemetry.rs

```diff
@@ -446,9 +438,16 @@ fn stats_report_key(op: Option<&opentelemetry::Value>, query: &str) -> String {
         tracing::warn!("Could not find required definition: {}", query);
         return GRAPHQL_UNKNOWN_OPERATION_NAME.to_string();
     }
-    tracing::debug!("looking for operation: {}", op_name);
+    tracing::debug!("looking for operation: {}", if op_name.is_empty() { "{empty}" } else { &op_name });
```

Comment:
I'm fine with using `{empty}` but just wonder if we should use `-` to be consistent with what we actually end up replacing the op_name with...

Reply:
👍  Tweaked in 7dc8302937ce408668d540d08116ec98eeebf7cb.